recipes/macfsevents/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "MacFSEvents" %}
+{% set version = "0.7" %}
+{% set sha256 = "95d3cddaf8a42435bfbd50087785ee9e3ebb8325242cfd06a88ea21f85bcb56f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not osx]
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
```

Comment:
nit: Very minor suggestion. Seems there is an extra space between `install` and the first argument. Would suggest cutting it to one space.